### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SuperCoac**
Short Description: 

Fixes:
- 
- 
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ This project uses Google Maps. There's one map coupled with the project, but as
       -m MODE, --mode MODE                          Set farming Mode for the bot ('all', 'poke', 'farm')
       -w SPEED,  --walk SPEED                       Walk instead of teleport with given speed (meters per second max 4.16 because of walking end on 15km/h)
       --distance_unit UNIT                          Set the unit to display distance in (e.g, km for kilometers, mi for miles, ft for feet)
-      --initial-transfer                            Start the bot with a pokemon clean up, keeping only the higher CP of each pokemon. It respects -c as upper limit to release.
+      --firsttrans                                  Start the bot with a pokemon clean up, keeping only the higher CP of each pokemon. It respects -c as upper limit to release.
```
Comment (solderzzc) [inline README.md]: hmm, why we need this changes?

Comment (jtdroste) [inline README.md]: +1 on that, we don't have that config option...

**Comment by jtdroste** [comment]
We currently don't have that config option...

Closing PR

**Comment by SuperCoac** [comment]
Sorry, didn't see that was changed

**Pull request closed without merging**